src/operator/c_lapack_api.h

```diff
@@ -70,8 +70,38 @@
 
 using namespace mshadow;
 
-// Will cause clash with MKL fortran layer headers
+#define XSTR(x) STR(x)
+#define STR(x) #x
+
+#pragma message "The value of MSHADOW_USE_MKL: " XSTR(MSHADOW_USE_MKL)
```

Comment:
what are theses `#pragma message` for?

Reply:
these I put to debug the error I am seeing in amalgamation builds.. but they dont help, so removing them